src/components/shopModal.html

```diff
@@ -4,6 +4,22 @@
         <div class="modal-content">
             <div class="modal-header text-center">
                 <h4 data-bind="text: player.town().name() + ' Shop'">Shop</h4>
+                <h4 style="display: inline;">
```

Comment:
Bootstrap has `d-inline` class, which may be better over inline styles :)

Reply:
Turns out it wasn't even needed